### PR TITLE
Minor change to unity SDK

### PR DESCRIPTION
**Pull request by nima-ap-zz**
Change a null return to string.Empty for better practice.

### CODE REVIEW

File: targets/unity-v2/make.js [hunk 1]
```diff
@@ -484,7 +484,9 @@ function getCustomApiLogic(tabbing, apiCall)
 {
     if (apiCall.name === "ExecuteFunction")
         return tabbing + "var localApiServerString = PlayFabSettings.LocalApiServer;\n" +
-            tabbing + "if (!string.IsNullOrEmpty(localApiServerString))\n" +
+            tabbing + "var debugFunctions = PlayFabSettings.LocalDebugFunctions;\n" + 
+            tabbing + "var shouldDebug = debugFunctions.Contains(\"*\") || debugFunctions.Contains(request.FunctionName);\n" +
```
Comment (MGudgin) [inline targets/unity-v2/make.js]: Equals rather than Contains for the first clause. Plus see comments on other PRs about whether we should do the specific functions thing.

File: targets/unity-v2/source/Shared/Internal/Util.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
+using System.Linq;
```
Comment (MGudgin) [inline targets/unity-v2/source/Shared/Internal/Util.cs]: Why did you need to add this?

File: targets/unity-v2/make.js (hunk 1 above)
Comment (MGudgin) [reply targets/unity-v2/make.js]: This comment applies to all the PRs. Just realized my disconnect here. Contains works, although I'd prefer it if we required "*" to be the only thing in the list. 

And I'm still not convinced we should do the specific functions, especially for private preview.

Comment (nima-ap-zz) [reply targets/unity-v2/make.js]: If it's done and it works why would we remove it?

Comment (nima-ap-zz) [reply targets/unity-v2/make.js]: @MGudgin waiting for your response before taking action on this.

**Review by MGudgin — APPROVED** [review]
